packages/react-tv-cli/scripts/tizen/run.js

```diff
@@ -0,0 +1,177 @@
+  const content = fs.readFileSync(appinfo, {encoding: 'utf-8'});
+
+  const re = new RegExp(/tizen:application id="(.*?)"/);
+  const matches = content.match(re);
+
+  if (!matches) {
+    return null;
+  }
+
+  return matches[1];
+}
+
+function isReactTVTizenProject(root) {
+  const appinfo = path.resolve(root, 'react-tv/tizen/config.xml');
+  if (fs.existsSync(appinfo)) {
+    return true;
+  }
+  return false;
+}
+
+function run(root) {
+  let tizen_CLI_ENV = process.env.TIZEN_CLI || false;
+  if (!tizen_CLI_ENV) {
+    tizen_CLI_ENV = defaultCLIEnv();
+  }
+
+  process.env.PATH = `${tizen_CLI_ENV}:${process.env.PATH}`;
+
+  if (!isReactTVTizenProject(root)) {
+    const msg = `This project isn\'t a React-TV Tizen Project:
+        Just run "react-tv init"`;
+    return console.log(chalk.dim('[react-tv]'), msg);
+  }
+
+  const packageJson = require(path.resolve(root, 'package.json'));
+  const ReactTVConfig = packageJson['react-tv'];
+  if (!ReactTVConfig) {
+    return console.log(
+      chalk.dim('[react-tv]'),
+      'You should set react-tv properties on package.json'
+    );
+  }
+
+  if (!ReactTVConfig.files || !ReactTVConfig.files.length) {
+    return console.log(chalk.dim('[react-tv]'), 'You should add files');
+  }
+
+  // TODO: option to create/select profiles?
+  const securityProfiles = execSync(
+    `${tizen_CLI_ENV}/tizen security-profiles list`
+  )
+    .toString()
+    .trim()
+    .split('\n');
+  if (!securityProfiles ||
+      securityProfiles[securityProfiles.length - 1].indexOf('[Profile Name]') > 0) {
+    return console.log(
+      chalk.dim('[react-tv]'),
+      'No tizen security profiles found'
+    );
+  }
+
+  // Select the last profile
+  const selectedProfile = securityProfiles[securityProfiles.length - 1]
+    .split(' ', 1)[0]
+    .trim();
+
+  const tizenPath = path.resolve(root, 'react-tv/tizen');
+
+  process.on('exit', cleanup);
+  process.on('SIGINT', cleanup);
+  process.on('SIGUSR1', cleanup);
+  process.on('SIGUSR2', cleanup);
+  process.on('uncaughtException', cleanup);
+
+  function cleanup() {
+    fs.removeSync(`${tizenPath}/icon.png`);
+    ReactTVConfig.files.forEach(file => {
+      fs.removeSync(`${tizenPath}/${file}`);
+    });
+  }
+
+  try {
+    cleanup();
+    fs.copySync(`${root}/react-tv/icon-large.png`, `${tizenPath}/icon.png`);
+
+    ReactTVConfig.files.forEach(file => {
+      const filePath = path.resolve(root, file);
+      const toFile = path.resolve(tizenPath, file);
+      fs.ensureDirSync(path.dirname(toFile));
+      fs.copySync(`${filePath}`, `${toFile}`);
+    });
+  } catch (e) {
+    return console.log('FAIL TO MOUNT', e.toString());
+  }
+
+  console.log('');
+  console.log(chalk.dim('Setting up Emulator...'));
+
+  const vms = execSync(
+    `${tizen_CLI_ENV}/../../emulator/bin/em-cli list-vm`
+  ).toString();
+
+  if (vms.indexOf('react-tv-tizen') < 0) {
+    execSync(`${tizen_CLI_ENV}/../../emulator/bin/em-cli create -n react-tv-tizen -p tv-samsung-4.0-x86`);
```

Comment:
For while I'm thinking about support `>=3` versions, even for renderer.

Reply:
For **real devices** the minimum supported version should be 2.3 I think.
If you look at the [TV Model Groups](http://developer.samsung.com/tv/develop/specifications/tv-model-groups) table you can see that a huge amount of TVs supports only 2.3 version.
Also the standard template created in Tizen Studio suggests to use 2.3 version (*see [config.xml](https://github.com/react-tv/react-tv/blob/64c4cbf650bba29cf3fe3d89ddaaa598e6878c7a/packages/react-tv-cli/bootstrap/react-tv/tizen/config.xml) from this PR* `required_version="2.3"`).
Moreover, almost every application from [Tizen apps exmples](https://github.com/SamsungDForum?language=javascript) specifies required version as 2.3.
And for the time being there're no phisical device in the wild with Tizen 4.0 Platform on board (only 2.3, 2.4, 3.0), so `>=3` is not correct excatly.

Anyway I think we can proceed with `>=3` for **emulators** since 3.0, 4.0 Plaforms can run 2.3 apps.

Reply:
Seems a reasonable thought, we can go on with `>=2.3`